KarhooUISDK/Screens/CheckoutScreen/Checkout/DetailsCellView.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  DetailsViewCell.swift
+//  KarhooUISDK
+//
+//  Created by Bartlomiej Sopala on 03/11/2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct DetailsCellView: View {
+    
+    @ObservedObject var model: DetailsCellModel
+    weak var delegate: DetailsCellViewControllerProtocol?
+
+    var body: some View {
+        Group {
```

Comment:
Please force light mode on this view for now. We'll circle back for dark mode.

Reply:
I can add `.colorScheme(.light)` modifier but as long as this view is inside of bigger view controller ( in this case CheckoutViewController) and we are forcing light mode for them there is no reason to force light mode on every single view.